data/human/ships.txt

```diff
@@ -2938,6 +2938,50 @@ ship "Scout"
 
 
 
+ship "Scrapper"
+	sprite "ship/scrapper"
+	thumbnail "thumbnail/scrapper"
+	attributes
+		category "Interceptor"
+		"cost" 60000
+		"shields" 450
+		"hull" 520
+		"required crew" 1
+		"bunks" 4
+		"mass" 62
+		"drag" 1.1
```

Comment:
```suggestion
		"drag" 1.4
```

This will still go massively faster than a shuttle (Overriding the nav system safeties, I'm guessing?), but not quite so blindly fast. 

As a comparison thing:
The Sparrow, with 176 mass, with 1 chipmunk plasma thruster, has a top speed of 766.
The Scrapper, with 186 mass, with 1 X2700 ion thruster, has a top speed of 752.

Having a heavier, pieced together junk heap of old ships, powered by a low performance ion engine, ship go almost as fast as a brand new fairly modern purpose-built interceptor with decently effective thrusters is... problematic... to me. 

Granted, the shuttle has more engine space than the sparrow, and I don't have a problem with the player being able to push it up to higher speeds than the sparrow. I just don't think it should be stock at such high speeds.

With a drag of 1.4 it is still a fair bit below the shuttle's 1.7, so it'll still be able to "break the safety margins" on the shuttle, but it's not quite as outrageous compared to brand new interceptors. Drag of 1.4 gives it a stock speed of 591, and with greyhound thrusters it can get up to 947 quite easily. I didn't test atomics, but those should have a max speed well over a thousand; so even with a drag of 1.4 it is still going to be a serious contender for racing against other interceptors, fighters, and small ships.